contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol

```diff
@@ -237,7 +238,7 @@ abstract contract BasicNFTOmnibridge is
 
         bytes memory data = _prepareMessage(_token, _receiver, _tokenIds, _values);
 
-        bytes32 _messageId = _passMessage(data, _isOracleDrivenLaneAllowed(_token, _from, _receiver));
```

Comment:
Please explain this

Reply:
By default `_isOracleDrivenLaneAllowed(_token, _from, _receiver)` always return true that mean oracle will automatically `executeSignature` that mint/unlock NFT on the foreign so oracle will pay a transaction fee. If we set `false` oracle will skip that so user can claim that

Reply:
OK. Please remain this code as commented out code and put the comment to here

Reply:
I've already done it